File: 1281. Subtract the Product and Sum of Digits of an Integer1.ts
function subtractProductAndSum(n: number): number {
    const digits = n.toString().split("");
    let prod = 1, sum = 0;
    for(let i = 0; i < digits.length; i++){
        prod *= Number(digits[i]);
        sum += Number(digits[i]);
    }
    return prod-sum;
};

File: 1281. Subtract the Product and Sum of Digits of an Integer1.cpp
class Solution {
public:
    int subtractProductAndSum(int n) {
        int sum = 0, prod = 1;

        while(n > 0){
            int digit = n%10;
            prod *= digit;
            sum += digit;
            n = n/10;
        }

        return prod-sum;
    }
};
